libminifi/include/utils/TimeUtil.h

```diff
@@ -35,15 +35,15 @@
 #include "StringUtils.h"
 
 // libc++ doesn't define operator<=> on durations, and apparently the operator rewrite rules don't automagically make one
-#if defined(_LIBCPP_VERSION) && _LIBCPP_VERSION < 16000
+#if defined(_LIBCPP_VERSION)
```

Comment:
According to cppreference, `operator<=>(duration, duration)` should exist in C++20, so this looks like a bug/not-yet-implemented feature.  Can we add a static assert here which will fail when `libc++` fixes/implements this?  Otherwise, we won't notice that this workaround can be removed.